airflow/models/__init__.py

```diff
@@ -4201,6 +4222,8 @@ def sync_to_db(self, owner=None, sync_time=None, session=None):
         orm_dag.owners = owner
         orm_dag.is_active = True
         orm_dag.last_scheduler_run = sync_time
+        orm_dag.default_view = self._default_view
```

Comment:
Can we test this in `tests.models:DagTest.test_sync_to_db`?

Reply:
Good one, added a test